examples/svelte/column-pinning/src/App.svelte

```diff
@@ -279,17 +236,15 @@
     {/if}
     <table class="border-2 border-black">
       <thead>
-        {#each isSplit ? $table.getCenterHeaderGroups() : $table.getHeaderGroups() as headerGroup}
+        {#each isSplit ? table.getCenterHeaderGroups() : table.getHeaderGroups() as headerGroup}
           <tr>
             {#each headerGroup.headers as header}
               <th colSpan={header.colSpan}>
                 <div class="whitespace-nowrap">
                   {#if !header.isPlaceholder}
-                    <svelte:component
-                      this={flexRender(
-                        header.column.columnDef.header,
-                        header.getContext()
-                      )}
+                    <FlexRender
```

Comment:
When I do this in my project with this adapter source imported I get

Type 'ColumnDefTemplate<HeaderContext<TData, unknown>> | undefined' is not assignable to type 'undefined'.
  Type 'string' is not assignable to type 'undefined'.ts(2322)

Reply:
If you could provide a minimal reproduction, that would be very helpful.